yt/frontends/halo_catalog/tests/test_outputs.py

```diff
@@ -88,7 +84,7 @@ def test_halo_catalog_boundary_particles(self):
         for field in ["particle_mass"]:
```

Comment:
looks like this test is a bit broken
```suggestion
        for field in fields:
```